packages/openapi-types/index.ts

```diff
@@ -7,17 +7,22 @@ export namespace OpenAPI {
   // }>
   export type Document<T extends {} = {}> =
     | OpenAPIV2.Document<T>
+    | OpenAPIV3.V30.Document<T>
```

Comment:
this is backwards incompatible.  can you instead add a new OpenAPIV31 namespace?

Reply:
Sure, I've prepared 3.1 types under another namespace, also checked types again and made some small changes for interfaces to allow overwriting its properties.